data/Govamarket.jason

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Clinton Chidera",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a frontend developer",
+ 
+  
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Your-user-name",
```

Comment:
@Govamarket you need to add your Twitter account's username.

Reply:
@Govamarket

Reply:
@Govamarket is my  Twitter account username.